### DISCUSSION TITLE
Ability to specify the node execution path in env

### DISCUSSION THREAD

**Discussion by vladyslav-mikhieiev** [spatie/mjml-php#15] [Ideas]
How about adding the ability to specify the node execution path via an environment variable. The current implementation in the project does not cover all nodejs installation options. As an example, the package will not work if nodejs is installed via nvm

**Comment by freekmurze** [comment]
Feel free to send a PR. Make sure to update docs as well

**Comment by vladyslav-mikhieiev** [comment]
PR #16
